phpstan.neon

```diff
@@ -0,0 +1,9 @@
+includes:
+    - ./vendor/nunomaduro/larastan/extension.neon
+
+parameters:
+    paths:
+        - src
+    level: 5
+    excludePaths:
+        - src/Assetic/*
```

Comment:
We need to get rid of the custom usage of Assetic in the first place

Reply:
Indeed, but this is in there for now until that happens.

Reply:
@bennothommo we can now get rid of this, although I'm not opposed to integrating Assetic with phpstan as well on its own repo.